### DISCUSSION TITLE
Todos

### DISCUSSION THREAD

**Discussion by NTDuck** [NTDuck/tomfoolery#3] [General]
1. Complete terminal-based monolith
a. Choose architecture (Clean Architecture + MVP), database (in-memory), and UI (terminal)
b. Complete architectural demo
c. Complete as per requirements: add more use cases, change UI, consider switching to different database (pending: SQLite)
d. Optimize e.g. opting for Trie (business logic, so stay at core/utils/)
e. Refactor
2. Complete graphical monolith
a. UI
b. Refactor
c. APIs and/or DBs
d. Refactor
e. Refactor
f. Refactor
g. Refactor
h. Refactor
i. Lethality Yorick

**Comment by NTDuck** [comment]
From the ground they come
to it they return

**Reply by FearOfTheSea** [reply]
I do not know what lies beyond. But it is better than this.
